pkg/storage/unified/migrations/service.go

```diff
@@ -49,34 +48,25 @@ func ProvideUnifiedStorageMigrationService(
 }
 
 func (p *UnifiedStorageMigrationServiceImpl) Run(ctx context.Context) error {
-	// TODO: temporary skip migrations in test environments to prevent integration test timeouts.
-	if os.Getenv("GRAFANA_TEST_DB") != "" {
-		return nil
-	}
-
 	// skip migrations if disabled in config
 	if p.cfg.DisableDataMigrations {
```

Comment:
```suggestion
	if p.cfg.EnableStorageMigrations {
```
Can we swap this so it is opt-in?

We will need much more support prep before this can get pushed live to on-prem deployments